src/html/test.pug

```diff
@@ -16,4 +16,4 @@ body
 
 	p
 		each icon in icons
-			i.simpleicons(title=icon.name, style=`color: #${icon.color}`)= icon.ligature
+			i.simpleicons.simpleicons--color(class=`simpleicons-${icon.cssClass}`, title=icon.name)
```

Comment:
Would be interesting to also have a toggle here that switches between the class variant and the ligature variant on top of light vs. dark.

While we're at that, I think it is also useful to have a toggle for the color (no color/just black vs. brand color) 🤔

Reply:
Sounds good, this turns it into more of an actual preview page rather than just some "hey it works" document.

Reply:
True, for something like this that is the best way (I know) to test if everything works